locales/nl.json

```diff
@@ -36,4 +36,5 @@
   "Show help": "Toon help",
   "Show version number": "Toon versie nummer",
   "Did you mean %s?": "Bedoelde u misschien %s?"
+  "Arguments %s and %s are mutually exclusive": "Argumenten %s en %s zijn onderling uitsluitend"
```

Comment:
looks like you're missing a `,` on the prior line, I think this is why there's the red-highlight. Could I bother you to also add translations for:

```
"Positionals:": "Positionals:",
"command": "command"
```

These new phrases have just been added.